bot/code_review_bot/__init__.py

```diff
@@ -100,6 +107,45 @@ def is_publishable(self):
 
         raise Exception("Unsupported publication mode {}".format(settings.publication))
 
+    def build_hash(self):
+        """
+        Build a uniquely identifying hash for that issue
+        """
+        assert self.revision is not None, "Missing revision"
+
+        try:
+            # Load all the lines affected by the issue
+            file_content = self.revision.load_file(self.path)
+        except requests.exceptions.HTTPError as e:
+            if e.response.status_code == 404:
+                logger.warning(
+                    "Failed to download a file with an issue", path=self.path
+                )
+                return
```

Comment:
What will the user of the function do in this case? Maybe we could still build the hash, but without the line content?

Reply:
Good idea, i'll try that